airbyte-webapp/src/locales/en.json

```
@@ -677,6 +678,31 @@
   "connectorBuilder.uiYamlToggle.yaml": "YAML",
   "connectorBuilder.resetAll": "Reset all",
   "connectorBuilder.emptyName": "(empty)",
+  "connectorBuilder.inputsTitle": "User inputs",
```

To be consistent with Global Configuration:
```suggestion
  "connectorBuilder.inputsTitle": "User Inputs",
```